src/derivatives/atlas.md

```diff
@@ -8,45 +8,43 @@ segmentations, and other knowledge annotations such as landmarks defined with
 respect to individual or template brains that are supported by spaces
 such as surfaces and regular grids (images).
 
-For derivatives with atlases in their provenance corresponding to individual subjects,
-the organization follows the standards for BIDS raw and derivatives.
+The organization follows the standards for BIDS raw and derivatives.
 The following entities MAY be employed to specify template- and atlas-derived results:
 
 -    [`tpl-<label>`](../glossary.md#template-entities) is REQUIRED to specify derivatives defining
-     a [template](../common-principles.md).
+     a [template](../common-principles.md).  MUST NOT be used alongside `atlas-<label>`
+-    [`atlas-<label>`](../glossary.md#atlas-entities) is REQUIRED to specify derivatives defining an [atlas](../common-principles.md).  MUST NOT be used alongside `template-<label>`
 -    [`space-<label>`](../glossary.md#space-entities) is REQUIRED to disambiguate derivatives defined with
      respect to different [coordinate systems](../appendices/coordinate-systems.md), following the general
-     BIDS-Derivatives specifications.
+     BIDS-Derivatives specifications.  It is also REQUIRED when [`tpl-<label>`](../glossary.md#template-entities) is used,
+     and the template is NOT serving as the authoritative definition of a space.
 -    [`cohort-<label>`](../glossary.md#cohort-entities) is REQUIRED to disambiguate derivatives defined with
-     respect to different cohort instances of a single [space (coordinate system)](../appendices/coordinate-systems.md).
+     respect to different cohort instances.
      Please note that [`cohort-<label>`](../glossary.md#cohort-entities) MUST NOT be used if neither
-     [`tpl-<label>`](../glossary.md#template-entities) nor [`space-<label>`](../glossary.md#space-entities) are used.
--    [`atlas-<label>`](../glossary.md#atlas-entities) is REQUIRED to encode files pertaining
-     or derived from the atlas identified by the entity's label.
+     [`tpl-<label>`](../glossary.md#template-entities) nor [`space-<label>`](../glossary.md#space-entities)
+     nor [`atlas-<labl>`](../glossary.md#atlas-entities) are used.
```

Comment:
Two changes happening here:

1- I don't think cohort interacts with atlas in a way that the two entities are necessary. We should recommend that a derivative with atlas- defined indicates the specific space and cohort in the JSON sidecar.

2- the atlas- entity definition was moved up, and, as I indicated, I think it is best to define template, space and cohort, and then move on to atlas, seg, etc.

```suggestion
     [`tpl-<label>`](../glossary.md#template-entities) nor [`space-<label>`](../glossary.md#space-entities) are used.
-    [`atlas-<label>`](../glossary.md#atlas-entities) is REQUIRED to encode files pertaining
     or derived from the atlas identified by the entity's label.
```

Details about (1):

The atlas entity doesn't need space and/or cohort definition (unless you are looking at very specific problems such as the agreement of a single atlas' regions projected from different templates in which that atlas has been defined, and for that I would use desc-):

```
sub-01/
    sub-01_atlas-HCP_dseg.json  <- RECOMMENDED to add here this comes from MNI152NLin6 and cohort if applies.
    sub-01_atlas-HCP_dseg.nii.gz
```

In the case of template definitions, atlas MUST have a cohort defined, if the template has cohorts:

```
tpl-MNIInfant/
    cohort-1/
        tpl-MNIInfant_cohort-1_atlas-InfantAtlas_dseg.nii.gz
```

This leaves a related edge-ish case (independent of atlas): when you want to use the space- in a derivative and you must also note the cohort. I'm summoning @mgxd and @effigies here. What we are doing in this case is:

```
sub-01/
    sub-01_space-MNIInfant+1_T1w.nii.gz
```

Using + as a concatenating token. Perhaps this is a good place in the BEP to incorporate that `+` is RECOMMENDED in the `space-` label when cohorts are involved.

Reply:
`+` is not yet accepted into BIDS. https://github.com/bids-standard/bids-specification/pull/1926